optimum/onnxruntime/modeling_ort.py

```diff
@@ -189,31 +210,31 @@ def forward(self, *args, **kwargs):
     @staticmethod
     def load_model(
         path: Union[str, Path],
-        provider: Optional[str] = "CPUExecutionProvider",
+        provider: str = "CPUExecutionProvider",
         session_options: Optional[ort.SessionOptions] = None,
-        provider_options: Optional[Dict] = None,
-        **kwargs
-    ):
+        provider_options: Optional[Dict[str, Any]] = None,
+    ) -> ort.InferenceSession:
         """
-        Loads an ONNX Inference session with a given provider. Default provider is `CPUExecutionProvider` to match the default behaviour in PyTorch/TensorFlow/JAX.
+        Loads an ONNX Inference session with a given provider. Default provider is `CPUExecutionProvider` to match the
+        default behaviour in PyTorch/TensorFlow/JAX.
 
-        Arguments:
-            path (`str` or `Path`):
+        Args:
+            path (`Union[str, Path]`):
```

Comment:
disagree, I don't see how it improves readability of the doc. The `or` is standard in transformers and I think it's better.